src/core/db/models.py

```diff
@@ -43,3 +59,64 @@ class Shift(Base):
 
     def __repr__(self):
         return f'<Shift: {self.id}, status: {self.status}>'
+
+
+class User(Base):
+    """Модель для пользователей"""
+    name = Column(String(100), nullable=False)
+    surname = Column(String(100), nullable=False)
+    date_of_birth = Column(DATE, nullable=False)
+    city = Column(String(50), nullable=False)
+    phone_number = Column(String(11), unique=True, nullable=False)
+    telegram_id = Column(BigInteger, unique=True, nullable=False)
+
+    def __repr__(self):
+        return f'<User: {self.id}, name: {self.name}, surname: {self.surname}>'
+
+    @validates('name', 'surname')
+    def validate_name_and_surname(self, key, value) -> str:
+        regex = "^[a-zа-яё ]+$"
+        if re.search(regex, value.lower()) is None:
+            raise ValueError('Фамилия или имя не корректные')
+        if len(value) < 2:
+            raise ValueError('Фамилия и имя должны быть больше 2 символов')
+        return value.title()
+
+    @validates('city')
+    def validate_city(self, key, value) -> str:
+        regex = "^[a-zA-Zа-яА-ЯёЁ -]+$"
+        regex_words = "[a-zA-Zа-яА-ЯёЁ]+"
+        if re.search(regex, value) is None:
+            if re.search(regex_words, value) is None:
```

Comment:
эти два ифа можно объеденить в один и проверять на None не обязательно, только перепроверь, а не просто переиспользуй прдложенный мной вариант
```suggestion
        if re.search(regex, value) is None &&  re.search(regex_words, value):
```